docs/release-notes.txt

```diff
@@ -6,6 +6,49 @@ Release Notes
 Version 0.12
 ------------
 
+Version 0.12.3
+^^^^^^^^^^^^^^
+
+**Release Date:** April 27, 2020
+
+- Add a tutorial for the new (experimental) Native API.
+
+- Add support for locally testing experiments via ``det e create --local``.
+
+- Add :class:`determined.experimental.Determined` class for accessing Experiment, Trial, and Checkpoint objects.
+
+- TensorBoard logs now appear under the ``storage_path`` for ``shared_fs`` checkpoint configurations.
+
+- Allow commands, notebooks, shells, and TensorBoards to be killed before they are scheduled.
+
+- Print container exit reason in trial logs.
+
+- Choose a better default for the ``--tail`` option of command logs.
+
+- Add additional REST API endpoints for trials.
+
+- Support a startup script inside agent docker images.
```

Comment:
That :point_up: reads like the startup script is added to a Docker image, which I don't believe is what [pull/243](https://github.com/determined-ai/determined/pull/243) accomplishes.
How about: `Support the execution of a startup script inside the agent docker container`?

Reply:
done